src/main/java/uk/gov/pay/webhooks/app/WebhookMessageSendingQueueProcessorConfig.java

```diff
@@ -29,14 +33,18 @@ public int getThreadDelayInMilliseconds() {
         return threadDelayInMilliseconds;
     }
 
-    public int getNumberOfThreads() {
-        return numberOfThreads;
+    public int getWebhookMessageSendingQueueProcessorNumberOfThreads() {
+        return webhookMessageSendingQueueProcessorNumberOfThreads;
     }
 
     public int getInitialDelayInMilliseconds() {
         return initialDelayInMilliseconds;
     }
 
+    public int getConnectionPoolNumberOfThreads() {
+        return connectionPoolNumberOfThreads;
```

Comment:
I think calling these "threads" though isn't quite right, it's essentially the maximum number of simultaneous http connections that can be available in the pool at any given time for a client to use, the client can access those connections from any thread.

I think something like "httpClientConnectionPoolSize" or something like that might better reflect it

Reply:
Updated